my_init.d/factory_reset.sh

```diff
@@ -1,34 +1,43 @@
-#!/bin/bash -e
-
-# Debugging.
-set -x
+#!/bin/bash -eu
 
 # Environment.
 export SHELL=/bin/bash
 
-# Performing factory reset of your AiiDAlab environment:
-# 0 - No reset (noop).
-# 1 - Remove locally installed software and apps (removes ~/apps/ and ~/.local/).
-# 2 - Remove all files and directories within the users home directory.
-
-
-case "${AIIDALAB_FACTORY_RESET}" in
-
-  0)
-    exit 0
-    ;;
-
-  1)
-    rm -rf "/home/${SYSTEM_USER}/apps"
-    rm -rf "/home/${SYSTEM_USER}/.local"
-    ;;
-
-  2)
-    find /home/${SYSTEM_USER}/ -mindepth 1 -delete
-    ;;
-
-  *)
-    echo "Unknown factory reset mode: ${AIIDALAB_FACTORY_RESET}"
-    exit 1
-    ;;
-esac
+# Bit mask for performing factory reset of the AiiDAlab environment:
+# 0b000 0 - No reset (noop).
+# 0b001 1 - Remove locally installed software and apps (removes ~/apps/ and ~/.local/).
+# 0b010 2 - Remove all files and directories within the users home directory.
+
+# If the ~/AIIDALAB_FACTORY_RESET file exists, parse it (and remove it).
+if [ -e "${HOME}/AIIDALAB_FACTORY_RESET" ]; then
+  RESET_MODE="`cat ${HOME}/AIIDALAB_FACTORY_RESET`"
+  rm -f "${HOME}/AIIDALAB_FACTORY_RESET"  # Remove file after parsing.
+fi
```

Comment:
```suggestion
# If the ~/AIIDALAB_FACTORY_RESET file exists, parse it (and remove it).
if [ -e "/home/${SYSTEM_USER}/AIIDALAB_FACTORY_RESET" ]; then
  RESET_MODE="`cat /home/${SYSTEM_USER}/AIIDALAB_FACTORY_RESET`"
  rm -f "/home/${SYSTEM_USER}/AIIDALAB_FACTORY_RESET"  # Remove file after parsing.
fi
```
The scripts from `my_init.d` folder are run by root, so using `${HOME}` doesn't make sense.